components/director/pkg/graphql/graphqlizer/graphqlizer.go

```diff
@@ -598,12 +598,18 @@ func (g *Graphqlizer) BundleCreateInputToGQL(in graphql.BundleCreateInput) (stri
 				{{- if $i}}, {{- end}} {{ EventDefinitionInputToGQL $e }}
 			{{- end }}],
 		{{- end }}
-		{{- if .Documents }} 
+		{{- if .Documents }}
 		documents: [
-			{{- range $i, $e := .Documents }} 
+			{{- range $i, $e := .Documents }}
 				{{- if $i}}, {{- end}} {{- DocumentInputToGQL $e }}
 			{{- end }} ],
 		{{- end }}
+		{{- if .CorrelationIDs }}
```

Comment:
Just a note if you need to create a bundle with the new correlation IDs input, in the tests, for example, you have to update the director component in the e2e tests go module. That's valid for other components as well.